hc/settings.py

```diff
@@ -136,7 +136,15 @@
 )
 COMPRESS_OFFLINE = True
 
+#send emails using Gmail.
 EMAIL_BACKEND = "djmail.backends.default.EmailBackend"
+DJMAIL_REAL_BACKEND = 'djmail.backends.async.EmailBackend'
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
```

Comment:
The code is now updated, Check commit 997ce1b0bf2cf0279211b729ae91fbd60bfe5176 to review